ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -10,6 +10,8 @@ import shared.stores 1.0 as SharedStores
 import shared.popups.keycard 1.0
 import shared.stores.send 1.0
 
+import StatusQ.Core.Theme 0.1
```

Comment:
Is this used ?

Reply:
Yes, that removes console warning for `Theme` when you get see "Keycard module is busy" popup.